### ISSUE TITLE
Bundle runtime prerequisistes with portable versions

### ISSUE THREAD

**Issue by slawo-ch** [TabularEditor/TabularEditor3#589]
Can we find a way to 'bundle' any runtime prerequisites with the portable versions or go back to an more commonly pre-installed runtime?
The whole point of a portable version is to not need admin privileges to run, and the installer - especially a runtime installer - requires that.

**Comment by otykier** [comment]
Agreed. We’ll should probably include the runtimes in the .zip. Going back is not an option due to some functionality in 3.3.0+ that requires .NET 6.

These are the prerequisites needed:

- [.NET 6 Desktop Runtime x64](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.6-windows-x64-installer)
- [VC++ 2015-2022 Redistributable x64](https://aka.ms/vs/17/release/vc_redist.x64.exe)

(and similar for x86)

**Comment by otykier** [comment]
@slawo-ch Starting from v. [3.3.3](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.3) we no longer require the VC++ 2015-2022 Redistributable, so the only prerequisite needed is the .NET 6 Desktop Runtime. If still needed, we may be able to provide a self-contained (portable) version of TE3 which includes the .NET 6 Desktop Runtime binaries, although this file would likely be +50 MB larger than the current portable version, which doesn't include the .NET 6 Desktop Runtime.

**Comment by slawo-ch** [comment]
The desktop runtime does not come preinstalled by default and likely won't ever as per: See: https://github.com/dotnet/core/issues/6411

Not including .NET 6 in the version makes it necessary to elevate to admin on installation and each lib update. 

50MB for a portable version that makes it standalone and portable is a low price to pay IMO.

Thank you :)

**Comment by otykier** [comment]
> The desktop runtime does not come preinstalled by default and likely won't ever as per: See: [dotnet/core#6411](https://github.com/dotnet/core/issues/6411)

Thanks for pointing my attention to that thread. Indeed it seems that an optional portable version of TE3 which includes the .NET Desktop Runtime (self-contained) is the way forward. We’ll make it happen!
